src/foundry/client/data/documents/playlist.d.mts

```diff
@@ -264,18 +265,24 @@ declare global {
        * A channel in CONST.AUDIO_CHANNELS where all sounds in this playlist are played
        * @defaultValue `"music"`
        */
-      channel: fields.StringField<{ choices: typeof foundry.CONST.AUDIO_CHANNELS; initial: string; blank: false }>;
+      channel: fields.StringField<{ choices: typeof CONST.AUDIO_CHANNELS; initial: string; blank: false }>;
 
       /**
        * The playback mode for sounds in this playlist
-       * @defaultValue `CONST.PLAYLIST_MODES.SEQUENTIAL`
+       * @defaultValue `CONST.PLAYLIST_MODES.SEQUENTIAL` (`0`)
```

Comment:
I mentioned it before but i just don't think `0` is that useful of an addition. Might've been in your stash though.